bloc-did-method.md

```diff
@@ -8,99 +8,165 @@ The `did:bloc` DID method allows groups of independent organizations to share cu
 
 Independent *stakeholder* organizations which wish to transact with one another using DIDs can come together to form a *consortium* to manage their shared custody of a ledger.
 
+The DID method operates as a discovery service that allows resolvers to discover the stakeholder organizations within a consortium, verify that the consortium itself is endorsed by the stakeholders, and access sidetree endpoints owned and exposed by the individual stakeholders, which the resolver can subsequently use to perform DID document operations within the consortium.
+
 Definitions:
 
-- Consortium: a group of Stakeholder organizations which all manage DIDs with their own Sidetree nodes, all on the same ledger.
-- Stakeholder organization (or just stakeholder): an independent organization which follows some cooperative rules with the other stakeholder organizations within the same consortium to allow them all to manage DIDs using their own Sidetree nodes on the same ledger. Stakeholders host ledger nodes, host sidetree endpoints, and sign policy changes.
+- Consortium: a group of Stakeholder organizations which all manage DIDs with their own Sidetree nodes, all on the same ledger. A Consortium hosts the configuration files which a resolver uses when first connecting to the network.
+- Stakeholder organization (or just stakeholder): an independent organization which follows some cooperative rules with the other stakeholder organizations within the same consortium to allow them all to manage DIDs using their own Sidetree nodes on the same ledger, and allow resolvers to manage DIDs through any of the stakeholders. Stakeholders host ledger nodes, host sidetree endpoints, and sign policy changes.
 
 #### `did:bloc` DID Format
 
 A DID for the Block DID Method has the following format:
 
-`did:bloc:[host]:[doc hash]`
+`did:bloc:[host]:[doc ID]`
+
+Where `[host]` is a URL host (eg, `foo.example.net`) and `[doc ID]` is the document ID as used in Sidetree.
```

Comment:
isn't `host` the consortium's hostname?

Reply:
eg. under "Consortium Data Model" there is the example `http://[consortium domain]/.well-known/did-bloc`. `[consortium domain]` is used there